zerver/lib/topic.py

```diff
@@ -82,6 +82,11 @@ def topic_match_sa(topic_name: str) -> "ColumnElement[bool]":
     return topic_cond
 
 
+def get_resolved_topic_condition() -> "ColumnElement[bool]":
```

Comment:
Let's add `_sa` to the name of this function, since it's a SQLAlchemy function and that's the pattern we're using for them here.